masterlist.yaml

```diff
@@ -6237,11 +6239,13 @@ plugins:
         display: '[SKSE](http://skse.silverlock.org)'
         condition: 'not checksum("rt_AutoHarvest.esp", BBDBA57A)'
   - name: 'RWB - Realistic Wolf Behavior.esp'
+     url: [ 'https://www.nexusmods.com/skyrim/mods/23785' ]
     dirty:
```

Comment:
Indentation is off by one space for the _url_ key.

![image](https://user-images.githubusercontent.com/1944639/67052384-eb9ce700-f135-11e9-8428-fbf00f67ea50.png)